### PR TITLE
PHX-149

### PR DESCRIPTION
**Pull request by hasnain-naveed**

### CODE REVIEW

**Comment by afzaledx** [comment]
:+1:

**Comment by chrisndodge** [comment]
Thanks team, I'll give this a try today and then merge it

**Comment by chrisndodge** [comment]
@srpearce since the rest of the doc team is on vacation can you look at the text that is in waiting_for_app_shutdown.html?

File: edx_proctoring/templates/proctored_exam/waiting_for_app_shutdown.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You have not yet submitted this proctored exam
```
Comment (srpearce) [inline edx_proctoring/templates/proctored_exam/waiting_for_app_shutdown.html]: How about this?

You are about to complete your proctored exam

**Comment by chrisndodge** [comment]
@afzaledx @hasnain-naveed merging. I just did a small change to help make a few sections a bit more readable. And also updated text based on @srpearce's recommentation. Thanks for the good work, sorry you had to do it twice.

**Pull request merged**